proto/workload.proto

```diff
@@ -1,7 +1,5 @@
 syntax = "proto3";
 
-option java_package = "io.spiffe.workloadapi.grpc";
```

Comment:
Good Catch Max. 
This doesn't require the *_pb files to be regenerated?
If it is not necessary but resulting files are different, them I'm in favor of generate the new ones and update the code.

Reply:
Sorry, missed that. It does change, doesn't affect anything but still worthwhile to keep them in sync. 
Thanks for the catch.